### ISSUE TITLE
BenchmarkSwitcher.RunAllJoined throws InvalidOperationException

### ISSUE THREAD

**Issue by hygehyge** [dotnet/BenchmarkDotNet#519]
When I create some benchmark classes and run them via BenchmarkSwitcher.RunAllJoined, it throws InvalidOperationException.
BenchmarkDotnet version is 0.10.9.

```csharp
    public class Bench1
    {
        [Benchmark]
        public void Test()
        {
            System.Console.WriteLine("Test");
        }
    }

    public class Bench2
    {
        [Benchmark]
        public void Test()
        {
            System.Console.WriteLine("Test");
        }
    }

    public class Bench3
    {
        [Benchmark]
        public void Test()
        {
            System.Console.WriteLine("Test");
        }
    }

    public class Program
    {
        static void Main(string[] args)
        {
            var switcher = new BenchmarkSwitcher(new[]
            {
                typeof(Bench1),
                typeof(Bench2),
                typeof(Bench3)
            });
            switcher.RunAllJoined(); //<-- this line throws Exception, it says 'sequence contains no elements'
        }
    }
```

**Comment by AndreyAkinshin** [comment]
Hey @hygehyge,

thanks for the report, the problem is fixed now, the fix will be released in v0.10.10.
Now you can use our nightly feed (version 0.10.9.271).

See
* https://ci.appveyor.com/project/dotnetfoundation/benchmarkdotnet/build/0.10.9.271
* http://benchmarkdotnet.org/Guides/NuGet.htm#nightly

**Comment by hygehyge** [comment]
Thanks for quick update.
I'll try nightly build later.
